.gitignore

```diff
@@ -39,5 +39,15 @@ gtda/externals/pybind11
 .pytest_cache/
 .hypothesis/
 
+# Latex
+*.aux
+*.bbl
+*.blg
+*.brf
+*.log
+*.pdf
+*.synctex.gz
+doc/theory/glossary_backup
+
 # development
-gtda/mapper/scratch/*
+tda/mapper/scratch/
```

Comment:
Why is this line changed? In fact, it should not be here at all?

Reply:
I don't think I touch this. It is as found. I think the mapper people should decide if this is necessary.

Reply:
@wreise @ammedmar this line is a remnant of work of mine and @lewtun from back in December. It has no role to play in the current state and should IMO be removed.